1차/1차/윤형진_2016년.py

```diff
@@ -0,0 +1,23 @@
+def solution(a, b):
+    answer = ''
+    year_dict = {1 : 31, 2 : 29, 3 : 31, 4 : 30, 5 : 31, 6 : 30, 7 : 31, 8 : 31, 9 : 30, 10 : 31, 11 : 30, 12 : 31 }
+    day_counter = 0
+    day = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"]
+    
+    for i in year_dict.keys() :
```

Comment:
굳이 스타일 대로 하자면 
```suggestion
    for month in year_dict.keys() :
```
이런식으로 바꿔야겠지만 뭐 워낙에 귀찮은일이니까요